src/content/blog/val-town-newsletter-23.mdx

```diff
@@ -0,0 +1,167 @@
+import Templates from "./val-town-newsletter-23/Templates.png";
+
+🥳 Happy 2025! Last year, our product grew from a JavaScript playground to a stable FaaS (Functions as a Service) platform that thousands of engineers use in personal and professional contexts. Some highlights from 2024:
+
+- Raised [our seed round](/blog/seed)
+- [Max](https://maxmcd.com/), [Jackson](https://jxnblk.com/), [Justin](https://just-be.dev/), [Sophie](https://sophiehouser.com/) and [Charmaine](https://www.charmaineklee.com/) joined our team
+- Our infra became [fast](https://github.com/val-town/val-town-product/discussions/113#discussioncomment-10813787) and stable
+- Launched [Townie](/blog/townie), a shockingly useful code assistant
+- 23k of you made 135k vals that've run 214m times
+
+We have big plans for 2025. By the end of this year, we aim to be a mature cloud platform that serious engineering organizations use for core infrastructure. Here's some of the highlights:
+
+- Projects – groups of vals, files folders – with forks & 3-ways merges (currently in private beta)
+- Scalable infra & usage-based billing
+- Team accounts for companies to collaborate
+- Improvements to Discovery, Search, Townie, Observability
+- Continuing to [fast-follow with LLMs](/blog/fast-follow/)
+- SOC 2
+
+We have two new additions to the Val Town team! [Sophie](https://sophiehouser.com/) is a designer, programmer, and artist, who just moved from Berlin to join us here in Brooklyn. She's best-known for [a viral app she made as a teenager](https://sophiehouser.com/tampon-run-), but most recently worked with Nike and Delivery Hero. [Charmaine](https://www.charmaineklee.com/) is a technical product manager, with a deep background in developer tools. She's worked at Snapchat, Unity Labs, and Microsoft. We're thrilled to have them both on board.
+
+## Val Projects Beta
+
+Vals are great for small projects, but what if you want to build something bigger?
+
+**Introducing Val Projects**, a new way to organize multiple vals, files, and folders into a single place. Where a val is like a GitHub Gist, a Val Project is like a GitHub Repo.
+
+<Image src={ProjectsExample} alt="Val Projects Example" />
+
+Projects support:
+
+- Static assets, like images, videos, css, html, etc
+- Multiple files and folders
+- Scoped environment variables (that code outside the project _can't_ access)
+
+Projects will soon support:
+
+- Branches, forking, pull requests, and three-way merges
+- Townie, with the ability to understand and edit multiple files
+
+We are building a new way to collaborate on code – without git or GitHub. That might sound crazy.
+But many any of the web's Pro tools had to build their own version control, including Figma, Google Docs, and Notion.
+We think that what we've come up with gives you 80% of the power of git with 20% of the complexity.
+And if you need git & GitHub, it should be easy to sync your Val Project with a git repo. More on this soon.
```

Comment:
@tmcw @maxmcd @charmaine - curious if y'all think I should delete this or rewrite it. It may not belong here